runners/direct-java/src/main/java/org/apache/beam/runners/direct/TransformExecutor.java

```diff
@@ -92,7 +92,7 @@ private TransformExecutor(
 
   @Override
   public void run() {
-    MetricsContainer metricsContainer = new MetricsContainer(transform.getFullName());
+    MetricsContainerImpl metricsContainer = new MetricsContainerImpl(transform.getFullName());
```

Comment:
I don't believe so, since a MetricsContainer returns counters, which do not allow extracting values.